docs/graphcache/custom-updates.md

```diff
@@ -5,8 +5,10 @@ order: 3
 
 # Custom Updates
 
-_Graphcache_ will attempt to automatically react to your mutations' and subscriptions' results
-but sometimes this isn't possible. While it will update all normalized entities that it finds in
+Every time a `subscription` triggeres or a `mutation` receives a response GraphCache will look at the response and see
+if it can get a key from the response. This key is by default derived from the `__typename` and `id`/`_id`, unless
```

Comment:
> This process is the same as for queries but instead of starting at the `Query` root type, it will start searching for keyable entities, where an object's `__typename` and `id` or `_id` fields (or a custom `keys` config) are provided, so it can write normalized entities to the cache.